CMakeLists.txt

```diff
@@ -218,14 +218,16 @@ endif()
 
 if(UNIX AND NOT APPLE)
     # Installation is currently Linux-only
-    install(TARGETS DwarfTherapist
-            RUNTIME DESTINATION bin)
     if (BUILD_PORTABLE)
+        install(TARGETS DwarfTherapist
+                RUNTIME DESTINATION dwarf-therapist/bin)
         install(DIRECTORY share/
-                DESTINATION share)
+                DESTINATION dwarf-therapist/share)
         install(FILES LICENSE.txt README.rst
-                DESTINATION share/doc)
+                DESTINATION dwarf-therapist/share/doc)
     else()
+        install(TARGETS DwarfTherapist
+                RUNTIME DESTINATION bin)
         install(DIRECTORY share/
```

Comment:
This achieves what I need, but is there a more preferred method?

Reply:
Don't change paths you don't understand. You are breaking dwarf therapist here. What you want is changing the install prefix.

Reply:
Testing showed it works, so I'll just use this locally I guess. No need to accept the pr if you have reasons to keep it as is